mmv1/third_party/terraform/tests/resource_artifact_registry_repository_test.go.erb

```diff
@@ -74,4 +118,20 @@ resource "google_artifact_registry_repository" "test" {
 }
 `, repositoryID)
 }
+
+func testAccArtifactRegistryRepository_create(repositoryID string, versionPolicy string) string {
+	return fmt.Sprintf(`
+resource "google_artifact_registry_repository" "test" {
+  provider = google-beta
+
+  repository_id = "%s"
+  location = "us-central1"
+  description = "post-update"
+  format = "MAVEN"
+  mavenConfig {
```

Comment:
It looks like the tests are failing due to this being in camelcase - since this is in a Terraform context, it should use snake_case.

```suggestion
  maven_config {
```

Reply:
Thanks, done!